site/content/en/docs/Getting started/linux.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Linux"
+linkTitle: "Linux"
+weight: 1
+description: >
+  How to install and start minikube on Linux.
+---
+
+## Installation
+
+{{% tabs %}}
+{{% tab "Direct" %}}
+
+Download and install minikube to /usr/local/bin:
+
+```shell
+ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
+   && sudo install minikube-linux-amd64 /usr/local/bin/minikube
+```
+{{% /tab %}}
+{{% tab "Debian/Ubuntu (apt)" %}}
+
+Download and install minikube:
```

Comment:
This is redudnent, it keeps getting repeated ! it is clear that it is downloading it from the header

Reply:
I disagree with this being redundant, it's repeated once per tab, only one of which is going to be read by a user. Also there needs to be text here, otherwise you end up with an awkward double box in the preview.